### DISCUSSION TITLE
Animation Queue Fix

### DISCUSSION THREAD

**Discussion by CritLoren** [alandtse/skyrim_vr_address_library#39] [Ideas]
**Link to mod**
https://www.nexusmods.com/skyrimspecialedition/mods/82395

**SKSE-based**
Yes

**Source Code**
https://github.com/ersh1/AnimationQueueFix

**Permissions to use source code (Nexus or GitHub)**
![https://www.nexusmods.com/skyrimspecialedition/mods/82395](https://user-images.githubusercontent.com/22298459/227661232-623f4a82-a5d2-4607-9b0f-37794970e8f8.png)

**No existing port (if you have a bug request, send it to the author please)**
No existing port, the current version kind of works in VR but has no considerations for VRIK and possibly some other inconsistencies under the hood. Let me know if you think this should still qualify as a bug report there however.

**Needs to use CommonLib (most likely). I haven't investigated a straight AddressLib only port yet.**
From the credits:
Ryan for the invaluable [CommonLibSSE](https://github.com/Ryan-rsm-McKenzie/CommonLibSSE), and the [CommonLibSSE plugin template](https://github.com/Ryan-rsm-McKenzie/ExamplePlugin-CommonLibSSE).
[powerofthree](https://www.nexusmods.com/skyrimspecialedition/users/2148728) for the [CommonLibSSE fork](https://github.com/powerof3/CommonLibSSE).
[CharmedBaryon](https://github.com/CharmedBaryon/) for [CommonLibSSE-NG](https://github.com/CharmedBaryon/CommonLibSSE-NG/).

**Comment by alandtse** [comment]
It's Ng and should just work. At least I've been using it. Are you sure it doesn't work in VR?

**Reply by CritLoren** [reply]
I haven't used it myself but reports in the comments indicate it's causing problems when paired with VRIK, particularly when it comes to holsters.

**Reply by alandtse** [reply]
Yah, I gave ersh the [info](https://github.com/ersh1/AnimationQueueFix/commit/99a9a777a92fd3b82dd23969e02423b60b1a2078) on how to make it work in VR. My view is it's already ported. The issues people are seeing are likely related to having DAR without atom's [DAR patch.](https://www.nexusmods.com/skyrimspecialedition/mods/80893)

**Reply by CritLoren** [reply]
it looked like some people were having the holsters issue despite the patch, but I'll update this whenever I end up being able to try it myself, disc can be closed for now
